src/components/editor/editor-pane/autocompletion/link-and-extra-tag.ts

```diff
@@ -0,0 +1,68 @@
+import { Editor, Hint, Hints, Pos } from 'codemirror'
+import moment from 'moment'
+import { getUser } from '../../../../redux/user/methods'
+import { findWordAtCursor, Hinter } from './index'
+
+const allowedChars = /[[]]*/
+const wordRegExp = /^(\[(.*])?)$/
+const allSupportedLinks = [
+  '[link text](https://example.com)',
+  '[reference]: https://example.com',
```

Comment:
If we would want to be equal to version 1.6:

```suggestion
  '[link text](https:// "title")',
  '[reference]: https:// "title"',
```

Reply:
done

Reply:
It looks like you removed the example.com but didn't add the title-attribute-placeholder. Is this intentional?

Reply:
I guess I did not realize that the link could have this too. just a moment

Reply:
done